Documentation clean-up (#517)
hyperledger-labs/fabric-private-chaincode: README.md

   chaincodes. A chaincode library acts as intermediary between the chaincode
   in the enclave and the peer. The chaincode enclave exposes the Fabric
   chaincode interface and extends it with additional support for state
   encryption, attestation, and secure blockchain state access. This

>    code is executed inside an SGX TEE.

Reviewer: ```suggestion
   code is executed inside an Intel SGX enclave.
```

Author: ok

1. *Enclave Endorsement validation:* The enclave endorsement validation
   complements the peer’s validation system and is responsible for
   validating transactions produced by a chaincode enclave. In
   particular, the validator checks that a transaction contains a

>    valid signature issued by a registered chaincode enclave. If the
>    validation is successful, it marks the transactions as valid and
>    hands it over to the ledger enclave, which crosschecks the decision
>    before it finally commits the transaction to the ledger. This code

Reviewer: ```suggestion
   valid signature issued by a registered chaincode enclave. If the
   validation is successful, it marks the transactions as valid and
   transforms the chaincode state updates to a Fabric read/writeset,
   which is applied during the validation-and-commit phase. This code
```

Author: good that you found that as it makes me realize there are lots of other references to ledger enclave which also have to be fixed ..

- The `fpc-cc-execution`([puml](docs/design/fabric-v2%2B/fpc-cc-execution.puml)/[png](docs/design/fabric-v2%2B/fpc-cc-execution.png)) diagram provides further detail of the execution phase of an FPC chaincode, focusing in particular on the `getState` and `putState` interactions with the Ledger and verification of state with the Ledger Enclave.
- The `fpc-validation`([puml](docs/design/fabric-v2%2B/fpc-validation.puml)/[png](docs/design/fabric-v2%2B/fpc-validation.png)) diagram describes the FPC-specific process of validation and revalidation using the Ledger Enclave.
- The `fpc-components`([puml](docs/design/fabric-v2%2B/fpc-components.puml)/[png](docs/design/fabric-v2%2B/fpc-components.png)) diagram shows the important data structures of FPC components and messages exchanged between components.

> - The detailed message definitions can be found as [protobufs](protos/fpc)

Reviewer: ```suggestion
- The detailed message definitions can be found as [protobufs](protos/fpc).
```

Author: k

- [Alessandro Sorniotti](https://github.com/ale-linux) (aso@zurich.ibm.com)

## Core Team FPC 1.0
- [Mic Bowman](https://github.com/cmickeyb) (mic.bowman@intel.com)
- [Marcus Brandenburger](https://github.com/mbrandenburger) (bur@zurich.ibm.com)
- [Jeb Linton](https://github.com/jrlinton) (jrlinton@us.ibm.com)
- [Michael Steiner](https://github.com/g2flyer) (michael.steiner@intel.com)

> - [Bruno Vavala](https://github.com/cmickeyb) (bruno.vavala@intel.com)

Reviewer 2: ehm...cough...cough...
